db.js

```diff
@@ -0,0 +1,16 @@
+const { Client } = require('pg');
```

Comment:
Were you thinking about using an ORM?

Reply:
I was just looking for npm package to go with postgreSQL, I wouldn't know how to connect to the database otherwise. I don't fully understand ORM concept or how to use it.